src/DIRAC/WorkloadManagementSystem/Service/PilotManagerHandler.py

```diff
@@ -152,6 +158,54 @@ def export_getPilotOutput(self, pilotReference):
         shutil.rmtree(ce.ceParameters["WorkingDirectory"])
         return S_OK(resultDict)
 
+    ##############################################################################
+    types_getRemotePilotOutput = [str]
+
+    def export_getRemotePilotOutput(self, pilotReference):
+        """
+        Get remote pilot log files.
+
+        :param str pilotReference:
+        :return: S_OK Dirac object
+        :rtype: dict
+        """
+
+        # get pilot stamp by pilot reference:
+        result = self.pilotAgentsDB.getPilotInfo(pilotReference)
+        if not result["OK"]:
+            self.log.error("Failed to get info for pilot", result["Message"])
+            return S_ERROR("Failed to get info for pilot")
+        if not result["Value"]:
+            self.log.warn("The pilot info is empty", pilotReference)
+            return S_ERROR("Pilot info is empty")
+
+        pilotDict = result["Value"][pilotReference]
+        pilotStamp = pilotDict["PilotStamp"]
+        owner = pilotDict["OwnerDN"]
+        group = pilotDict["OwnerGroup"]
+        vo = getVOForGroup(group)
+
+        if self.loggingPlugin is None:
+            result = ObjectLoader().loadObject(
+                f"WorkloadManagementSystem.Client.PilotLoggingPlugins.{self.configValue}", self.configValue
+            )
+            if not result["OK"]:
+                self.log.error("Failed to load LoggingPlugin", f"{self.configValue}: {result['Message']}")
+                return result
+
+            componentClass = result["Value"]
+            self.loggingPlugin = componentClass()
+            self.log.info("Loaded: PilotLoggingPlugin class", self.configValue)
+
+        res = self.loggingPlugin.getRemotePilotLogs(pilotStamp, vo)
+
+        if res["OK"]:
+            res["Value"]["OwnerDN"] = owner
+            res["Value"]["OwnerGroup"] = group
```

Comment:
What is the use of these values?

Reply:
Considering that in https://github.com/DIRACGrid/DIRAC/pull/7157 I have removed OwnerDN, do not use it here.

Reply:
OK